src/Rule.js

```diff
@@ -0,0 +1,72 @@
+function Rule(){
+    this.findError = [];
+}
+Rule.prototype.isRequired = function(){
+    this.findError.push(function(value){
+        let isValid;
+        (value == undefined || typeof(value) == null) ? isValid = false : isValid = true
```

Comment:
Can be simplified into: let isValid = (value == undefined || typeof(value) == null)